Source/bmalloc/bmalloc/TZoneHeapManager.cpp

```diff
@@ -141,7 +158,8 @@ void TZoneHeapManager::init()
 
     m_state = TZoneHeapManager::Seeded;
 
-    atexit(dumpRegisterdTypesAtExit);
+    if (verbose)
```

Comment:
I think we can use `constexpr` here?

Reply:
Actually, we can't.  With constexpr, the compiler will eliminate the call to atexit(dumpRegisterdTypesAtExit); and then complain that dumpRegisterdTypesAtExit() is unused.